frontend/src/assets/content/mthood/cutting-instructions/helpful-information.md

```diff
@@ -1,6 +1,6 @@
 * Tools you might want to consider bringing with you include a handsaw to cut your tree; gloves to protect your hands; a tarp to sit on and/or to move your tree once it's cut; and rope or straps to secure your tree to your vehicle.
-* Be sure another tree has space to grow within 8 feet of the one you plan to cut. Choosing a tree from an overcrowded area will give the remaining trees more space to grow.
-* Be aware of where the tree will fall and ensure all members of your party (and your vehicle) are out of the path of the falling tree.
+* Be sure another tree has space to grow within 8 feet of the one you plan to cut. Choosing a tree from an dense forested area will give the remaining trees more space to grow.
```

Comment:
should this be densely forested?

Reply:
Yes, I suppose "densely forested" would be accurate. Is that I change I can do here some how?